backend/apis/events/get-event.api.js

```diff
@@ -62,7 +62,7 @@ function getEventById(eventId, cbk, connection) {
       WHERE events.id = ? AND isDeleted = false;
 
       SELECT
-        leadEvents.leadId, leads.gender, leads.leadStatus
+        leadEvents.leadId, leads.gender, leads.leadStatus, leads.firstName, leads.lastName
```

Comment:
This API was already getting all the leads for the event. I just needed to add their first and last names.